packages/protocol/contracts/package.json

```diff
@@ -0,0 +1,10 @@
+{
+    "name": "@celo/contracts",
+    "version": "1.0.0",
+    "author": "cLabs",
+    "license": "LGPL-3.0",
+    "dependencies": {
+        "openzeppelin-solidity": "^2.5.0",
```

Comment:
you probably want to use a fixed version here. using dynamic version for contracts seems scary